example/index.php

```diff
@@ -3,8 +3,11 @@
 
 use AfricasTalking\SDK\AfricasTalking;
 
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
+$dotenv->load();
+
 $username = "sandbox";
-$apiKey = getenv("API_KEY");
+$apiKey = $_ENV["API_KEY"];
```

Comment:
Suggestion;

Could this be named `AFRICASTALKING_API_KEY`? 

`API_KEY` is rather generic and I could be having a similarly named key in my application.

Reply:
your suggestion is correct.